README.md

```diff
@@ -0,0 +1,50 @@
+# TREF - Text Representation Evaluation Framework
```

Comment:
What is the origin of this name :D?

Reply:
it is just an idea, wdyt? do you have any propositions?

Reply:
I think it should be same as our package name to not confuse users. And therefore all path should be changed e.g. `embeddings` module should fit our destination name e.g. `tref`